src/modules/endpoint/endpoints/jsonTypes/JsonCommonTypes.js

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
We need to come up with a way of structuring our types. So in this case we really need types because we apply them to untyped data.
My proposal would be to create a `types.js` file in each module/route and just throw all of them in there. Thats how a lot of npm packages are doing it.
Or we create a `types` folder and structure the types there accordingly. For example: `src/modules/endpoint/types/json.js`

Reply:
We will have the same problem with the type of our state. Not sure where to put it right now in the react-native prototype.

In the webapp we do not type the state and actions.

Reply:
You mean our redux state?